backends/tfhe-cuda-backend/cuda/src/fft/twiddles.cu

```diff
@@ -1,6 +1,6 @@
 #include "cuComplex.h"
 
-__constant__ double2 negtwiddles[4096] = {
```

Comment:
We've been wondering for a long time if this could indeed bring performance gains. Have you confirmed that?

Reply:
Yup, between 12-18% improvement in the 64-bit operations. 
Some results:
div: 3.6s -> 3.04s
mul: 378ms ->333ms
add: 52.4ms ->45.6ms
if_then_else: 21ms -> 18.4ms

Reply:
Awesome!